### ISSUE TITLE
iOS build does not work

### ISSUE THREAD

**Issue by DonghwanKIM0101** [homuler/MediaPipeUnityPlugin#529]
### Plugin Version or Commit ID

v0.8.2

### Unity Version

2020.3.32f1

### Your Host OS

macOS Monterey Version 12.3

### Target Platform

iOS

### [iOS Only] XCode Version

13.3.1

### Build Command

python build.py --desktop cpu --ios arm64 -vv

### Bug Description

I want to build in other platform (m1 Mac, Windows, iOS), but they all show same problem.

In iOS, the program shows only black screen about 1 second, and it's killed.

In m1 Mac and Windows, the program shows only black screen.

### Steps to Reproduce the Bug

python build.py build --desktop cpu --ios arm64 -vv

### Log

In Xcode, I cannot find any logs...

### Screenshot/Video

<img width="1512" alt="Screen Shot 2022-04-15 at 10 37 03 PM" src="https://user-images.githubusercontent.com/65400489/163577325-43c020be-6f4e-4f8f-be75-6c58e8e12f08.png">

**Comment by homuler** [comment]
> In Xcode, I cannot find any logs...

Please attach [LogFiles](https://docs.unity3d.com/Manual/LogFiles.html).

**Comment by DonghwanKIM0101** [comment]
In Xcode, there wasn't any logs. (When I run the app, it is killed soon and the log window is also closed.)

BTW, I solve the problem.
I did two things but I don't know what is the exact solution, so anyone with similar problems just refer to it.

1. `python build.py clean`
2. I uncheck "Copy only when installing" option to be checked in Build Phase > Embed Frameworks.
In the first time I built iOS application in Xcode, I found other errors to install application on iPhone and some blogs guided me to check this option. However, I uncheck it after `python build.py clean`, I can run the app successfully.
